core/src/main/antlr4/com/databricks/labs/remorph/parsers/tsql/TSqlParser.g4

```diff
@@ -2802,6 +2803,7 @@ querySpecification
     ;
 
 selectOptionalClauses
+    // TODO: Fix ORDER BY; it needs to be outside the set operations instead of between.
```

Comment:
And also this - sometimes the original author implemented it just as reading the syntax diagram from MS, which does not always deal with such things